modules/logging-es-deploy-cli.adoc

```diff
@@ -0,0 +1,319 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/cluster-logging-deploying.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-es-deploy-cli_{context}"]
+= Installing {logging-uc} with Elasticsearch using the CLI
```

Comment:
It should be highlighted that:
  - Elasticsearch is deprecated
  - Elasticsearch Operator is not supported in 4.16 for Openshift Logging, then, these instructions shouldn't be present in 4.16 and they should be removed or leaving the section but making reference to don't supported for OCP 4.16 and newer versions